### PR TITLE
use custom image

### PR DESCRIPTION
**Pull request by pweil-**
for use with "imageResourceGroup": "opstest",  "imageName": "openshift-gi-1521144906"

Fixes https://github.com/kargakis/acs-engine/issues/23

### CODE REVIEW

File: parts/openshift/master.sh [hunk 1]
```diff
@@ -1,92 +1,19 @@
...
-baseurl=https://mirror.openshift.com/enterprise/enterprise-3.7/v3.7.23-1_2018-01-11.2/RH7-RHAOS-3.7/x86_64/os/
-enabled=1
-gpgcheck=0
-sslclientcert=/var/lib/yum/client-cert.pem
-sslclientkey=/var/lib/yum/client-key.pem
-EOF
-
-yum -y install atomic-openshift-master atomic-openshift-node atomic-openshift-sdn-ovs dnsmasq etcd openshift-ansible-roles openvswitch
-
-cat >/etc/cni/net.d/80-openshift-network.conf <<'EOF'
```
Comment (pweil-) [inline parts/openshift/master.sh]: per networking folks this gets deleted and recreated when sync with master occurs. No need to lay it down

**Comment by pweil-** [comment]
@jim-minter 

```
[root@master 0]# curl -k https://0.0.0.0:8443/version
{
  "major": "",
  "minor": "",
  "gitVersion": "v1.9.1+a0ce1bc657",
  "gitCommit": "a0ce1bc",
  "gitTreeState": "clean",
  "buildDate": "2018-02-26T02:50:15Z",
  "goVersion": "go1.9.2",
  "compiler": "gc",
  "platform": "linux/amd64"
}

[root@master 0]# oc get nodes
NAME      STATUS    ROLES     AGE       VERSION
master    Ready     <none>    1m        v1.9.1+a0ce1bc657
node1     Ready     <none>    54s       v1.9.1+a0ce1bc657
```

Containers are stuck in creating, looking at that next but master and nodes are 'up' with your stock config in 3.9 with the image mentioned in the description.  Deployment took 7 minutes 28 seconds.  There appears to be an issue when resolving the hostname to an ip.  For a while it seems to timeout on an origin and resolve on the next attempt and then, all of a sudden, it starts working and everything finishes.

I wonder if that's affecting the rhel base image too.  You can check by running `host master`.  For a time I see:

```
[root@master 0]# host master
;; connection timed out; trying next origin
master.mpg1emjlvuxu1feo5r222udohf.bx.internal.cloudapp.net has address 10.0.0.10
```

File: parts/openshift/master.sh [hunk 2]
```diff
@@ -1,92 +1,21 @@
...
-{
-  "cniVersion": "0.2.0",
-  "name": "openshift-sdn",
-  "type": "openshift-sdn"
-}
-EOF
+iptables -A OS_FIREWALL_ALLOW -p tcp -m state --state NEW -m tcp --dport 8443 -j ACCEPT
```
Comment (jim-minter) [inline parts/openshift/master.sh]: add a # TODO

File: parts/openshift/master.sh [hunk 3]
```diff
@@ -1,92 +1,21 @@
...
+iptables-save > /etc/sysconfig/iptables
+systemctl restart iptables
 
+# todo add this to the image
```
Comment (jim-minter) [inline parts/openshift/master.sh]: note the 172.30.0.0/16 also may end up needing to be configurable assuming we enable integration with existing vnets

File: parts/openshift/master.sh [hunk 4]
```diff
@@ -118,13 +47,14 @@ oc adm registry
...
 oc adm router
 
-for file in /usr/share/ansible/openshift-ansible/roles/openshift_examples/files/examples/v3.7/db-templates/*.json \
-    /usr/share/ansible/openshift-ansible/roles/openshift_examples/files/examples/v3.7/image-streams/*-rhel7.json \
-	  /usr/share/ansible/openshift-ansible/roles/openshift_examples/files/examples/v3.7/quickstart-templates/*.json \
-	  /usr/share/ansible/openshift-ansible/roles/openshift_examples/files/examples/v3.7/xpaas-streams/*.json \
-	  /usr/share/ansible/openshift-ansible/roles/openshift_examples/files/examples/v3.7/xpaas-templates/*.json; do
-	oc create -n openshift -f $file
-done
+# TODO openshift-ansible-roles not currently installed in custom image
```
Comment (jim-minter) [inline parts/openshift/master.sh]: would prefer to see this PR merge after openshift-ansible-roles goes in so the commenting is avoided, but not a hard block

Comment (pweil-) [reply parts/openshift/master.sh]: don't want to merge this until your config work is done so we can test it in conjunction with that.

Comment (pweil-) [reply parts/openshift/master.sh]: and I should say, until a new image is available as well...no point in regressing

**Comment by jim-minter** [comment]
closing for #42

**Pull request closed without merging**